elastic/kibana
# [Synthetics] quote monitor name to prevent invalid yaml

## Summary

Appropriately quotes monitor names for synthetics integration policies, to ensure that customers can use monitor names that would otherwise break yaml.

### Testing
1. Create a private location
2. Save a monitor with name `[Synthetics] test` to that private location
3. Navigate to the agent policy for that location. Confirm that the integration policy was added to the agent policy, and that the name is correct
